tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs

```diff
@@ -252,7 +252,10 @@ public void Builds_NetCore31App_UsingNetCore31_DotNetSdkVersion()
             var appDir = volume.ContainerDir;
             var appOutputDir = "/tmp/NetCoreApp31MvcApp-output";
             var script = new ShellScriptBuilder()
-                .AddBuildCommand($"{appDir} -o {appOutputDir} --platform dotnet --platform-version 3.1.8")
```

Comment:
Let me know if we were purposely hardcoding these versions, and don't want them to increase as we add support for newer versions